src/main/java/com/ita/if103java/ims/mapper/dto/AssociateDtoMapper.java

```diff
@@ -20,6 +20,7 @@ public Associate toEntity(AssociateDto dto) {
             associate.setPhone(dto.getPhone());
             associate.setAdditionalInfo(dto.getAdditionalInfo());
             associate.setType(dto.getType());
+            associate.setActive(dto.isActive());
```

Comment:
I do not think this is good idea.

Please check If this change doesn't introduce ability to create/update **inactive** associate.

If you add this just for unit tests it is error pron